src/components/projects-summary/projects-summary.html

```diff
@@ -88,4 +88,25 @@ <h2 class="secondary">Resources</h2>
       </ul>
     </div>
   </div>
+  <div ng-if="$ctrl.recentlyViewedItems.length">
+    <h2 class="primary">Recently Viewed</h2>
```

Comment:
In the design doc it looks like the same size (perhaps even bigger) than "My Projects" http://openshift.github.io/openshift-origin-design/web-console/1-homepage/project-panel#recently-viewed
Pls see picture under 'Projects List (More than 2)"  @serenamarie125 ?  -thanks